src/devTools.js

```diff
@@ -257,8 +257,29 @@ function handleChange(state, liftedState, maxAge) {
   }
 }
 
+/*
+ * Get hostname for React Native if hostname is `localhost`
+ * For Android Emulator, the real localhost is `10.0.2.2` (Genymotion: 10.0.3.2)
+ */
+function getHostForRN(hostname) {
+  if (hostname !== 'localhost' || hostname !== '127.0.0.1') {
+    return hostname;
+  }
+
+  let host = hostname;
+  try {
```

Comment:
Maybe better to just have
```js
if (
  typeof window !== 'undefined' && window.__fbBatchedBridge &&
  window.__fbBatchedBridge.RemoteModules && window.__fbBatchedBridge.RemoteModules.ServerHost
) {
  return window.__fbBatchedBridge.RemoteModules.ServerHost.split(':')[0];
} else {
  return hostname;
}
```

Your implementation looks cleaner, but it's considered good practice to avoid try-catch where it is possible as it doesn't allow V8 to do optimizations ([more details](https://github.com/petkaantonov/bluebird/wiki/Optimization-killers)).

Also maybe better to move this in a separate module like `./utils/reactNative.js`?